code/game/objects/machinery/minelayer.dm

```diff
@@ -46,41 +46,41 @@
 		playsound(loc, 'sound/machines/twobeep.ogg', 25, 1)
 		return
 	var/turf/target_turf = pick_n_take(list_of_turfs)
-	if(target_turf.density || turf_block_check(src, target_turf) || locate(/obj/item/explosive/mine) in range(1, target_turf) || !line_of_sight(loc, target_turf))
+	if(target_turf.density || turf_block_check(src, target_turf) || locate(/obj/item/mine) in range(1, target_turf) || !line_of_sight(loc, target_turf))
 		throw_mine(list_of_turfs)
 		return
-	var/obj/item/explosive/mine/mine_to_throw = new /obj/item/explosive/mine(loc)
+	var/obj/item/mine/mine_to_throw = new /obj/item/mine/claymore(loc)
```

Comment:
This converts any type of mine to claymores?

Reply:
When I changed this only the claymore existed. I'll need to make a list to accommodate all the new mine types but does this item need to exist? I think it should just be deleted, it's so useless.